game/scripts/npc/abilities/zuus_cloud.txt

```diff
@@ -52,4 +55,3 @@
       }
     }
   }
-}
```

Comment:
This should not be removed. The file has unbalanced brackets because of this. Fixing the issue seems to make it not affected by CDR. Though it also seems to make the Nimbus die in one hit from a hero. Probably needs code in the modifier to do the hits-to-kill thing.